semantic-ruby/src/Language/Ruby/Tags.hs

```diff
@@ -6,6 +6,7 @@
 {-# LANGUAGE TypeApplications #-}
 {-# LANGUAGE TypeOperators #-}
 {-# OPTIONS_GHC -Wno-unrecognised-pragmas #-}
+{-# OPTIONS_GHC -Wno-incomplete-uni-patterns #-}
```

Comment:
Both of these are suspect— 🔥 both of em

Reply:
cool, `{-# HLINT ignore "Reduce duplication" #-}` is ok?